org.lflang/src/org/lflang/generator/cpp/CppExtensions.kt

```diff
@@ -1,13 +1,7 @@
 package org.lflang.generator.cpp
 
 import org.eclipse.emf.ecore.resource.Resource
-import org.lflang.InferredType
-import org.lflang.TargetProperty
-import org.lflang.TimeUnit
-import org.lflang.indexInContainer
-import org.lflang.isBank
-import org.lflang.isGeneric
-import org.lflang.isMultiport
+import org.lflang.*
```

Comment:
I believe we don't allow wildcard imports in our style guide.

Reply:
For Kotlin we refer to the [Coding Conventions](https://kotlinlang.org/docs/coding-conventions.html) which don't specify this. Wild card imports are quite idiomatic in Kotlin and are applied automatically by the IntelliJ auto-formatter.